testing/test_regressions.py

```diff
@@ -97,6 +100,52 @@ def test_case_mismatch_on_windows_git(tmp_path: Path) -> None:
     assert res is not None
 
 
+def test_fallback_message(wd: WorkDir, monkeypatch: pytest.MonkeyPatch) -> None:
+    """Check that no warning/error messages are normally outputted"""
+    # setup a basic project that would not trigger any (other) warning/errors
+    pyproject = """\
```

Comment:
Is there a fixture to more easily set all of this up?

Reply:
not yet , there has not been much work on e2e testing the build backend integrations